packages/ui/src/validators/hooks/useStakingStatistics.tsx

```diff
@@ -19,44 +19,51 @@ export const useStakingStatistics = () => {
     [api?.isConnected]
   )
 
-  const now = useObservable(() => api?.query.timestamp.now(), [api?.isConnected])
-  const totalIssuance = useObservable(() => api?.query.balances.totalIssuance(), [api?.isConnected])
-  const currentStaking = useObservable(
+  const [now, setNow] = useState(Date.now())
+  setInterval(() => setNow(Date.now()), 1000)
```

Comment:
This shouldn't be here. It will cause the page to re-render maybe every seconds but maybe a lot more than that because the interval is set on every render and never cleaned